Makefile.build

```diff
@@ -262,6 +273,10 @@ __config-sync-manifest-oss: __config-sync-manifest
 	rsync \
 		.output/manifests/acm-psp.yaml ${OSS_MANIFEST_STAGING_DIR}/acm-psp.yaml
 
+# Deprecated alias of build-manifests. Remove this once unused.
```

Comment:
What's this still used by?

Reply:
There are some CI jobs that use it. They'll be moved to the new target and this target can be removed once it's unused